### PR TITLE
v4 docs - Change incorrect LESS references to SCSS

### PR DESCRIPTION
**Pull request by davidjrichardson**

### CODE REVIEW

File: docs/content/typography.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Bootstrap includes simple and easily customized typography for headings, body te
 Bootstrap sets basic global display, typography, and link styles. Specifically, we:
 
 - Set `background-color: #fff;` on the `<body>`
-- Use the `@font-family-base`, `@font-size-base`, and `@line-height-base` attributes as our typographic base
-- Set the global link color via `@link-color` and apply link underlines only on `:hover`
+- Use the `$font-family-base`, `$font-size-base`, and `$ine-height-base` attributes as our typographic base
```
Comment (cvrebert) [inline docs/content/typography.md]: Lost the "l" in "line" here

File: docs/content/typography.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ Bootstrap includes simple and easily customized typography for headings, body te
...
+- Set the global link color via `$link-color` and apply link underlines only on `:hover`
 
-These styles can be found within `scaffolding.less`.
+These styles can be found within `_variables.scss`.
```
Comment (cvrebert) [inline docs/content/typography.md]: `_reboot.scss` is where the styles that make use of these variables live

**Comment by hnrch02** [comment]
Merged as https://github.com/twbs/bootstrap/commit/42792682acc780b8ea6bf56655293cf9919a9115.

**Comment by cvrebert** [comment]
Derp. Thanks @hnrch02

**Pull request closed without merging**